WorkflowFunctionAdminBundle/DataFixtures/MongoDB/LoadGroupDemoData.php

```diff
@@ -19,6 +19,7 @@ class LoadGroupDemoData extends AbstractLoadGroupData implements OrchestraProduc
     public function load(ObjectManager $manager)
     {
         $this->addRole('group2');
+        $this->addRole('group-admin');
```

Comment:
I don't have the begining of your work so I must ask you a question to validate this PR: does this group is already defined ?